examples/darwin-framework-tool/commands/clusters/ReportCommandBridge.h

```diff
@@ -237,7 +276,88 @@ class SubscribeEvent : public ModelCommand {
     chip::Optional<bool> mKeepSubscriptions;
     chip::Optional<bool> mAutoResubscribe;
     chip::Optional<chip::EventNumber> mEventNumber;
+    chip::Optional<bool> mIsUrgent;
     bool mSubscriptionEstablished = NO;
     uint16_t mMinInterval;
     uint16_t mMaxInterval;
+
+    void Shutdown() override
+    {
+        mSubscriptionEstablished = NO;
+        ModelCommand::Shutdown();
+    }
+
+    bool DeferInteractiveCleanup() override { return mSubscriptionEstablished; }
+
+private:
+    chip::ClusterId mClusterId = chip::kInvalidAttributeId;
```

Comment:
Why are we setting a cluster id to an attribute id?

Reply:
It changes for mClusterId to be set to kInvalidClusterId.